server/server.js

```diff
@@ -8,8 +8,12 @@ app.use(parser.json());
 app.use(parser.urlencoded({ extended: false }));
 app.use(express.static(`${__dirname}/../client/dist`));
 
-app.get('/', function (req, res) {
-  res.send(200);
+app.get('/rooms', function (req, res) {
```

Comment:
We need to fix this part before we can ship it. The `'/rooms'` endpoint here should be `'/rooms/:roomId'`

Reply:
RESOLVED!